src/components/Register/Register.js

```diff
@@ -0,0 +1,77 @@
+import axios from "axios";
+import { useState } from "react";
+import "../Login/login.css";
+
+export const Register = () => {
+  const [name, setName] = useState();
+  const [username, setUsername] = useState();
+  const [password, setPassword] = useState();
+  const [message, setMessage] = useState();
+
+  const registerClickHandler = async () => {
+    console.log(name, username, password);
+    try {
+      const response = await axios.post(
+        "http://localhost:5000/auth/signup",
```

Comment:
I think you forget to change the server URL here.

Reply:
yes! my bad, will change it, thanks for pointing it out!